### DISCUSSION TITLE
用aria2下载的时候总是提示磁盘满了

### DISCUSSION THREAD

**Discussion by Lo-kin** [lifegpc/bili#85] [Q&A]
![QQ图片20210418204547](https://user-images.githubusercontent.com/79306993/115146787-9d4e0880-a08a-11eb-9324-bd248e137c8b.png)
这个该怎么解决，求解答

**Comment by walwan** [comment]
难道不是真的磁盘满了吗？似乎是在preallocate的阶段提示磁盘满了诶。

**Reply by Lo-kin** [reply]
![QQ图片20210419194521](https://user-images.githubusercontent.com/79306993/115231149-ced8d980-a147-11eb-9e94-c33de453a47e.png)
G:盘还有好多空间啊

**Comment by lifegpc** [comment]
禁用aria2c吧。看看用内置的能否下载。

**Reply by Lo-kin** [reply]
是硬件的问题，已解决，谢谢了
